showCrime/showCrime/settings.py

```diff
@@ -1,3 +1,12 @@
+"""
+django app for https://OakCrime.org
+"""
+
+__author__ = "rik@electronicArtifacts.com"
+__credits__ = ["clinton.blackburn@gmail.com","actionspeakslouder@gmail.com"]
```

Comment:
The document you linked to is quite old, and I don't think it was written with respect to a collaborative open source project. I don't know of any project that lists credits in every file. Instead, there is typically an `AUTHORS` or `CONTRIBUTORS` file at the root of the repo (e.g. https://github.com/edx/course-discovery/blob/master/AUTHORS). Other projects forego the file altogether given the commit history tracks who contributed to the project.